src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java

```diff
@@ -29,28 +29,39 @@ public void setUp() {
     }
 
     @Test
-    public void execute_newPerson_success() {
+    public void execute_newTutor_success() {
         Tutor validTutor = new TutorBuilder().build();
-        Student validStudent = new StudentBuilder().build();
 
         Model expectedModelTutor = new ModelManager(model.getAddressBook(), new UserPrefs());
-        Model expectedModelStudent = new ModelManager(model.getAddressBook(), new UserPrefs());
         expectedModelTutor.addTutor(validTutor);
-        expectedModelStudent.addStudent(validStudent);
 
         assertCommandSuccess(new AddCommand(validTutor, PersonType.TUTOR), model,
                 String.format(AddCommand.MESSAGE_SUCCESS_TUTOR, validTutor), expectedModelTutor);
-        assertCommandSuccess(new AddCommand(validTutor, PersonType.STUDENT), model,
-                String.format(AddCommand.MESSAGE_SUCCESS_TUTOR, validStudent), expectedModelStudent);
     }
 
     @Test
-    public void execute_duplicatePerson_throwsCommandException() {
+    public void execute_newStudent_success() {
+        Student validStudent = new StudentBuilder().build();
+
+        Model expectedModelStudent = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModelStudent.addStudent(validStudent);
+
+        assertCommandSuccess(new AddCommand(validStudent, PersonType.STUDENT), model,
+                String.format(AddCommand.MESSAGE_SUCCESS_STUDENT, validStudent), expectedModelStudent);
+    }
+
+    @Test
+    public void execute_duplicateTutor_throwsCommandException() {
         Tutor tutorInList = model.getAddressBook().getTutorList().get(0);
```

Comment:
Can make use of predefined `INDEX_FIRST_PERSON` here:
```suggestion
        Tutor tutorInList = model.getAddressBook().getTutorList().get(INDEX_FIRST_PERSON.getZeroBased());
```